content/desktop/release-notes.md

```diff
@@ -25,6 +25,73 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+
+### Upgrades
+
+- [Docker Compose v2.27.0](https://github.com/docker/compose/releases/tag/v2.27.0)
+- [Docker Engine v26.0.1](https://docs.docker.com/engine/release-notes/26.0/#2601)
+- [Wasm](../desktop/wasm/_index.md) runtimes:
+  - Updated `runwasi` shims to `v0.4.0`
+  - Updated `deislabs` shims to `v0.11.1`
+  - Updated `spin` shim to `v0.13.1`
+- [Docker Scout CLI v1.8.0](https://github.com/docker/scout-cli/releases/tag/v1.8.0)
+- Docker Debug `v0.0.29`
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Improved Enhanced Container Isolation (ECI) security when running `docker build` commands in rootless containers.
+- Fixed a bug where `docker events` exited with `Unexpected EOF` when Docker Desktop entered/exited Resource Saver mode.
+- Fixed a bug where `docker stats --no-stream` hanged when Docker Desktop was in Resource Saver mode.
+- Fixed a bug in the self-diagnose CLI that incorrectly showed the VM had not started. Fixes [docker/for-mac#7241](https://github.com/docker/for-mac/issues/7241).
+- Fixed a bug where high-throughput port forward transfers could stall. Fixes [docker/for-mac#7207](https://github.com/docker/for-mac/issues/7207).
+- Fixed CLI-plugin symlinks not being removed when CLI apps were removed.
+- Fixed a bug in the shared ports drawer to show the right message for local engines.
+- Dev Environments is being sunset and has moved to the **Beta** tab in **Features in development**.
+
+#### For Mac
+
+- Fixed a bug where a segmentation fault was triggered with Virtualization Framework, on post-2019 Macs. See [docker/for-mac#6824](https://github.com/docker/for-mac/issues/6824).
+- Enabled `CONFIG_SECURITY=y` kernel config, for example for [Tetragon](https://tetragon.io/). Fixes [docker/for-mac#7260](https://github.com/docker/for-mac/issues/7250).
+- Re-added support for `SQUASHFS` compression. Fixes [docker/for-mac#7260](https://github.com/docker/for-mac/issues/7260).
+- Fixed a bug that caused a new version of Docker Desktop to be marked as damaged.
+- Increased network MTU when using qemu on Apple Silicon.
+- Fixed a bug preventing Docker Desktop to start if Rosetta was not installed.
+
+#### For Windows
+
+- Added a simplified provisioning mode for WSL2 that avoids the need for the ancillary `docker-desktop-data` WSL distribution (experimental).
+- Fixed bash completions for the Docker CLI in a WSL environment.
+- Fixed a regression in Docker Desktop 4.28 that caused host files bind-mounted into containers to not show up properly inside the container, when using Docker-in-Docker (via mounts of `/var/run/docker.sock`) on WSL.
+- Fixed a bug that would cause the following error `merging settings: integratedWslDistros type mismatch`.
+
+
+### Known issues
+
+#### For Windows
+
+- When starting Docker Desktop as a non admin user and as a member of docker-users group, the following error `connect ENOENT \\.\pipe\errorReporter` might be triggered. This can
```

Comment:
@lorenrh / @aevesdocker the condition that triggers that error is, user is a non admin user but not a member of docker-users group. Could you edit accordingly please. Thank you.

Reply:
@chelnak please review.